### ISSUE TITLE
End-User Post reply & edit ticket will repeat when reload/refresh the page 

### ISSUE THREAD

**Issue by ffsab00070875** [osTicket/osTicket#5234]
Dear all,

I'm using 1.14.1. (PHP 7, Ubuntu 18)

It is weird or something wrong when end-user post reply.
And the "Message Posted Successfully" always show in page.
So I reload the page, but the reply will show again (reply again). 
![擷取](https://user-images.githubusercontent.com/18123688/69775428-aa3b3500-11d3-11ea-99ff-44b546871feb.JPG)

Thanks!

**Comment by JediKev** [comment]
@ffsab00070875 

Of course, when you reload the page you’re replaying the POST request. We have a pull request to address this issue but it’s outdated needs to be refactored:
- https://github.com/osTicket/osTicket/pull/4509

Cheers.

**Comment by ffsab00070875** [comment]
@JediKev 

Dear JediKev,

How can I solve this issue or OsTicket Support Team will refactor?

BRs

**Comment by slima** [comment]
+1

**Comment by mrkylegp** [comment]
+1 for this. It's still an issue in v1.18.1
